test/multiverse/suites/rails_prepend/newrelic_prepender/newrelic_prepender.gemspec

```diff
@@ -13,4 +13,5 @@ Gem::Specification.new do |spec|
     f.match(%r{^(test|spec|features)/})
   end
   spec.require_paths = ["lib"]
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
I'm curious -- why add this here?

Also, have you done any absorbing of this gem? I'm somewhat skeptical about its contemporary relevance.

Reply:
There's a new cop, `Gemspec/RequireMFA`, that insists on this line being present in gemspec files. I don't know if we're ready to start using that declaration yet for our 2 main gemspec files, so I have disabled the cop for now and left those files untouched. But for this file that is basically a fixture file, I figured it was ok to leave the `rubocop -a` driven addition.